.storybook/main.ts

```diff
@@ -0,0 +1,15 @@
+import type { StorybookConfig } from "@storybook/nextjs";
+
+const config: StorybookConfig = {
+  stories: ["../src/**/*.stories.@(ts|tsx)"],
+  addons: ["@storybook/addon-essentials", "@chromatic-com/storybook", "@storybook/addon-interactions"],
```

Comment:
question(non-blocking): Are we using chromatic? Given we're pushing for Percy, is it worth standardising?

[There is a standard plugin for it](https://github.com/percy/percy-storybook#readme).